lib/feature/add_character/add_character_page.dart

```diff
@@ -0,0 +1,204 @@
+              onPrimary: Colors.grey[800], // цвет текста кнопки
+              elevation: 8,
+              shape: RoundedRectangleBorder(
+                borderRadius: BorderRadius.circular(16),
+              ),
+            ),
+            child: const Text('Add Character'),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildNeumorphicTextField(
+      TextEditingController controller, String labelText) {
+    return Container(
+      decoration: BoxDecoration(
+        color: Colors.grey[200], // Цвет фона текстового поля
+        borderRadius:
+            BorderRadius.circular(16), // Скругление углов текстового поля
+        boxShadow: [
+          BoxShadow(
+            color: Colors.grey[300]!, // Цвет тени
+            offset: Offset(5, 5), // Смещение тени по X и Y
+            blurRadius: 10, // Радиус размытия тени
+            spreadRadius: 1, // Размах тени
+          ),
+          BoxShadow(
+            color: Colors.white, // Цвет второй тени (белая)
+            offset: Offset(-5, -5), // Смещение второй тени по X и Y
+            blurRadius: 10, // Радиус размытия второй тени
+            spreadRadius: 1, // Размах второй тени
+          ),
+        ],
+      ),
+      child: TextField(
+        controller: controller,
+        decoration: InputDecoration(
+          labelText: labelText,
+          contentPadding: const EdgeInsets.symmetric(horizontal: 16),
+          border: InputBorder.none, // Убираем границы текстового поля
+        ),
+      ),
+    );
+  }
+
+  Widget _buildSuccessUI(BuildContext context, Character character) {
+    return Center(
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          const Icon(
+            Icons.check_circle,
+            color: Colors.green,
+            size: 50,
+          ),
+          const SizedBox(height: 10),
+          const Text(
+            'Character added!',
+            style: TextStyle(fontSize: 18, color: Colors.green),
+          ),
+          // show added items
+          Text('Title: ${character.title}'),
+          Text('Image URL: ${character.imageUrl}'),
+          Text('Description: ${character.description}'),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildErrorUI(BuildContext context, String error) {
+    return Center(
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          const Icon(
+            Icons.error,
+            color: Colors.red,
+            size: 50,
+          ),
+          const SizedBox(height: 10),
+          Text(
+            'Error: $error',
+            style: const TextStyle(fontSize: 18, color: Colors.red),
+          ),
+          const SizedBox(height: 10),
+          ElevatedButton(
+            onPressed: () {
+              context
+                  .read<AddCharacterCubit>()
+                  .addCharacter(title: '', imageUrl: '', description: '');
```

Comment:
is it okay that we don't have parameters here?

Reply:
Done
Added ones)